sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -88,6 +135,18 @@ export class ChangeFeedResponse<T> {
     readonly statusCode: number;
 }
 
+// @public
+export enum ChangeFeedStartFrom {
+    // (undocumented)
+    Beginning = 0,
```

Comment:
In JS, is there a recommendation on the order of values? Should the `0` be the default? If so, should it be `Now`?

Reply:
No order of values that I am aware of but have made Now as the default if no ChangeFeedStartFrom is passed.